docs/source/exit-codes.rst

```diff
@@ -0,0 +1,19 @@
+Exit codes
+==========
+
+=========  ============================================================================ ============================
+Exit code  Meaning                                                                      Applies to subcommand
+=========  ============================================================================ ============================
+0          Success                                                                      All
+1          Error in query sequence(s), BLAST options, or ElasticBLAST configuration     Submit
+2          Error in BLAST database                                                      Submit
+3          Error in BLAST engine                                                        Submit, status
+4          Out of memory                                                                Submit, status
```

Comment:
Is this supported?

Reply:
It appears that isn't the case. I removed it.